src/chess_piece_move_validation_tests/king/king.ino

```diff
@@ -0,0 +1,80 @@
+/**
+ * @file king.ino
+ * @author Ines Rohrbach, Nico Schramm
+ * @brief Test for ECPKing isMoveValid method.
+ * @version 0.1
+ * @date 2024-11-26
+ * 
+ * @copyright Copyright (c) 2024
+ * 
+ */
+
+#include <Dezibot.h>
+#include <EmbeddedChessPieces.h>
+#include <Wire.h>
+
+#define BAUD_RATE 9600
+
+Dezibot dezibot = Dezibot();
+
+void setup() {
+    Serial.begin(BAUD_RATE);
+    dezibot.begin();
+    delay(500);
+}
+
+void loop() {
+    Serial.println("\n=== STARTING TEST ===\n");
+
+    Serial.println("Testing king...");
+    testKing();
+
+    Serial.println("\n=== TEST DONE ===");
+    Serial.println("Sleeping for 10 seconds...\n");
+
+    delay(10000);
+}
+
+void testKing() {
+    const ECPChessField initialField = { F, 5 };
+    ECPKing king = ECPKing(dezibot, initialField, true);
+
+    // valid moves
+    Serial.println("Testing valid moves...");
+    ECPChessField validFields[] = {
+        { E, 6 }, { F, 6 }, { G, 6 },
+        { E, 5 }, /* fig */ { G, 5 },
+        { E, 4 }, { F, 4 }, { G, 4 },
+    };
```

Comment:
Please move `};` to the line above for consistency.

Reply:
Fixed in 6ae4f97418193f6045f94624918fad9938d50a31